server/database/models/user.ts

```diff
@@ -0,0 +1,40 @@
+import { Schema, model } from 'mongoose';
+
+const user = new Schema({
+  name: {
+    type: String,
+    required: true,
+  },
+  email: {
+    type: String,
+    required: true,
+  },
+  password: {
+    type: String,
+    required: true,
+  },
+  cv: {
+    type: String,
+    required: false,
+  },
+  bio: {
+    type: String,
+    required: false,
+  },
+  profile_picture: {
+    type: String,
+    required: false,
+  },
+  level: {
+    type: Number,
+    required: false,
+  },
+  role: {
+    type: String,
+    required: true,
+  },
+});
+
+const userSchema = model('user', user);
```

Comment:
Name all the models with the first letter in the capital, like User instead of user